Since the tests analyze the output messages, force the default locale.

http://bugs.gentoo.org/99766

--- tests/test_script.in
+++ tests/test_script.in
@@ -3,6 +3,19 @@
 # Test script for e2fsck
 #
 
+# NLS nuisances.
+for as_var in \
+  LANG LANGUAGE LC_ADDRESS LC_ALL LC_COLLATE LC_CTYPE LC_IDENTIFICATION \
+  LC_MEASUREMENT LC_MESSAGES LC_MONETARY LC_NAME LC_NUMERIC LC_PAPER \
+  LC_TELEPHONE LC_TIME
+do
+  if (set +x; test -z "`(eval $as_var=C; export $as_var) 2>&1`"); then
+    eval $as_var=C; export $as_var
+  else
+    unset $as_var
+  fi
+done
+
 if test "$1"x = x ; then
   TESTS=`ls -d $SRCDIR/[a-zA-Z]_* | $EGREP -v "\.failed|\.new"`
 else
